bug: zola deploy

on https://github.com/gabrielkoerich/orch/actions/runs/23988822612/job/69965310604:

```
Run zola --root docs build
Building site...
ERROR Failed to build the site
ERROR Couldn't find front matter in `/home/runner/work/orch/orch/docs/content/posts/morning-review-2026-04-04.md`. Did you forget to add `+++` or `---`?
Error: Process completed with exit code 1.
```

--- a/docs/content/posts/morning-review-2026-04-04.md
+++ b/docs/content/posts/morning-review-2026-04-04.md
@@ -1,3 +1,9 @@
++++
+title = "Morning Review — 2026-04-04"
+date = 2026-04-04
+description = "High agent success rates, multiple bug fixes, and operational health review"
++++
+
 # Morning Review - 2026-04-04
 
 ## Recent Commits & Progress

